tests/tensor/rewriting/test_basic.py

```diff
@@ -1298,41 +1299,60 @@ def test_local_join_make_vector():
 
 
 def test_local_sum_make_vector():
+    # To check that rewrite is applied, we must enforce dtype to
+    # allow rewrite to occur even if floatX != "float64"
     a, b, c = scalars("abc")
+    point = {a: 1.0, b: 2.0, c: 3.0}
     mv = MakeVector(config.floatX)
-    output = mv(a, b, c).sum()
-
-    output = rewrite_graph(output)
-    between = vars_between([a, b, c], [output])
-    for var in between:
-        assert (var.owner is None) or (not isinstance(var.owner.op, MakeVector))
+    output = mv(a, b, c).sum(dtype="float64")
+    rewrite_output = rewrite_graph(output)
+    expected_output = cast(
+        add(*[cast(value, "float64") for value in [a, b, c]]), dtype="float64"
+    )
+    # check rewrite
+    assert equal_computations([expected_output], [rewrite_output])
+    # check logic
+    np.testing.assert_almost_equal(output.eval(point), rewrite_output.eval(point))
 
-    # Check for empty sum
+    # Empty axes should return input vector since no sum is applied
     a, b, c = scalars("abc")
+    point = {a: 1.0, b: 2.0, c: 3.0}
     mv = MakeVector(config.floatX)
     output = mv(a, b, c).sum(axis=[])
+    rewrite_output = rewrite_graph(output)
+    # check rewrite
+    assert isinstance(rewrite_output.owner.op, MakeVector)
```

Comment:
Last nitpick is to also use equal_computation with expected output here

Reply:
Done!